sdk/src/wallet/account/operations/output_claiming.rs

```diff
@@ -123,6 +121,26 @@ where
         );
         Ok(output_ids_to_claim.into_iter().collect())
     }
+}
+
+impl<S: 'static + SecretManage> Account<S>
+where
+    crate::wallet::Error: From<S::Error>,
+{
+    /// Get basic and nft outputs that have
+    /// [`ExpirationUnlockCondition`](crate::types::block::output::unlock_condition::ExpirationUnlockCondition),
+    /// [`StorageDepositReturnUnlockCondition`] or
+    /// [`TimelockUnlockCondition`](crate::types::block::output::unlock_condition::TimelockUnlockCondition) and can be
+    /// unlocked now and also get basic outputs with only an [`AddressUnlockCondition`] unlock condition, for
+    /// additional inputs
+    pub async fn claimable_outputs(&self, outputs_to_claim: OutputsToClaim) -> crate::wallet::Result<Vec<OutputId>> {
+        log::debug!("[OUTPUT_CLAIMING] claimable_outputs");
```

Comment:
Maybe unnecessary

Reply:
Maybe

Reply:
Because already logged in account_details.claimable_outputs I meant

Reply:
I know, but could be interesting to know what has been called in what order etc :trollface:
Whatever, removed because now with this PR this place is fine